<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPresets</class>
 <widget class="QDialog" name="DialogPresets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pbClose">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>320</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="grFolder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>Folder</string>
   </property>
   <widget class="QListWidget" name="lwFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>231</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbFolderRename">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Rename Folder</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/48x48/folder-edit.png</normaloff>:/icons/48x48/folder-edit.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pbFolderNew">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Add Folder</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/48x48/folder-plus.png</normaloff>:/icons/48x48/folder-plus.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pbFolderDelete">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Delete Folder</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/48x48/folder-remove.png</normaloff>:/icons/48x48/folder-remove.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grPresets">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>261</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>Presets</string>
   </property>
   <widget class="QListWidget" name="lwPreset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>241</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPresetSave">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Save</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/48x48/content-save-settings.png</normaloff>:/icons/48x48/content-save-settings.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPresetDelete">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Delete</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/48x48/delete-circle.png</normaloff>:/icons/48x48/delete-circle.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPresetFavorite">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Favorite</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/48x48/heart.png</normaloff>:/icons/48x48/heart.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPresetRename">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Rename</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/48x48/pencil.png</normaloff>:/icons/48x48/pencil.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPresetSaveAs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Save as...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
